mipengine/controller/algorithm_executor.py

```diff
@@ -128,24 +128,24 @@ def run(self):
                 "One of the nodes participating in the algorithm execution "
                 "stopped responding"
             )
-            ctrl_logger.getLogger(__name__).error(
+            ctrl_logger.getAppLogger().error(
                 f"{error_message} \n{err=}"
             )  # TODO logging..
 
             raise AlgorithmExecutionException(error_message)
         except:
             import traceback
 
-            ctrl_logger.getLogger(__name__).info(f"{traceback.format_exc()}")
+            ctrl_logger.getAppLogger().info(f"{traceback.format_exc()}")
         finally:
             self.clean_up()
 
     def clean_up(self):
         # TODO logging..
-        ctrl_logger.getLogger(__name__).info(f"----> cleaning up global_node")
+        ctrl_logger.getAppLogger().info(f"----> cleaning up global_node")
         self.global_node.clean_up()
         for node in self.local_nodes:
-            ctrl_logger.getLogger(__name__).info(f"----> cleaning up {node:}")
+            ctrl_logger.getAppLogger().info(f"----> cleaning up {node:}")
```

Comment:
somewhere at the beginning
```
logger = ctrl_logger.getLogger(__name__)
```
and then anytime the logger is used 
``` 
logger.info(f"{message}")
```
and the same for all the other modules that use the logger..

Reply:
This is not implemented this way because in the next PR that the App (Request) Logger will require a context id, it won't be possible to initialize it at the beginning of the module.

The `context_id` will need to be provided when initializing the logger so the developer of the module will need to handle it.

Reply:
ok then you could initialize it in the init of the AlgorithmExecutor, the context_id is available in there
```
class AlgorithmExecutor:
    def __init__(
        self,
        algorithm_execution_dto: AlgorithmExecutionDTO,
        nodes_tasks_handlers_dto: NodesTasksHandlersDTO,
    ):
        self._context_id = algorithm_execution_dto.context_id
        
        ...
        self._logger=ctrl_logger.getAppLogger(self._context_id)
```

Reply:
Fixed